src/journal.sh

```diff
@@ -678,7 +678,11 @@ rljClosePhase(){
                             "($name)"
     local logfile="$(mktemp)"
     tail -n +$((__INTERNAL_PHASE_TXTLOG_START+1)) "$__INTERNAL_BEAKERLIB_JOURNAL_TXT" > $logfile
-    rlReport "$(echo "${name//[^[:alnum:]]/-}" | tr -s '-')" "$result" "$score" "$logfile"
+
+    # Replace all non-alphanumeric characters with dashes in the test name.
+    # Also make sure the test name does not start/end with dashes and ensure
+    # the dashes are not repeated.
+    rlReport "$(echo "${name//[^[:alnum:]]/-}" | sed 's/^-\+//' | sed 's/-\+$//' | tr -s '-')" "$result" "$score" "$logfile"
```

Comment:
we could simplify whole pipeline here
```suggestion
    rlReport "$(echo "${name//[^[:alnum:]]/-}" | sed -r 's/-+/-/g;s/^-//;s/-$//')" "$result" "$score" "$logfile"
```

Reply:
Thanks for the suggestion! Simplified. Marking as resolved.